dynamiqs/smesolve/sme_solver.py

```diff
@@ -15,24 +15,27 @@ def __init__(
         self,
         *args,
         jump_ops: Tensor,
-        meas_ops: Tensor,
         etas: Tensor,
         generator: torch.Generator,
         t_meas: Tensor,
     ):
-        self.meas_ops = meas_ops  # (len(meas_ops), n, n)
-        self.etas = etas  # (len(meas_ops))
-        self.generator = generator
         self.t_meas = t_meas  # (len(t_meas))
-
         super().__init__(*args, jump_ops=jump_ops)
```

Comment:
Can you rename `jump_ops` to `L` like you did in mesolve (I think?)?

Reply:
`SMESolver` inherits from `MESolver` so it's done automatically.